app/views/comments/_comments.html.slim

```diff
@@ -0,0 +1,13 @@
+  #comments
+    h2 Comments
+
+    - commentable.comments.each do |comment|
+      hr(id="#{dom_id comment}")
+      h5 = "#{ user_for_comment(comment) }, #{ l(comment.created_at, format: :short) }".html_safe
+      p = render_markdown(comment.text).html_safe
+
+    = simple_form_for new_comment, remote: true do |f|
+      = f.input :commentable_id, as: :hidden
+      = f.input :commentable_type, as: :hidden
+      = f.input :text
+      = f.submit 'Add comment'
```

Comment:
I'd add a `disable_with` here _(probable need to switch the input to a button then)_, users will probably hit the button multple times if there's no direct feedback and I currently don't see an option to delete a comment again.